pages/common/psgrep.md

```diff
@@ -0,0 +1,15 @@
+# psgrep
+
+> Search through processes with grep.
+
+- Find process lines containing a specific string:
+
+`psgrep {{'foo'}}`
```

Comment:
Are the single quotes necessary ? And I would rather have process_name than foo to match with the pgrep page.

Reply:
It doesn't appear that the quotes are necessary.  I can remove them.
`process_name` is fine with me.